chain/client/src/info.rs

```diff
@@ -135,13 +135,12 @@ impl InfoHelper {
         let me = client.validator_signer.as_ref().map(|x| x.validator_id());
         if let Ok(num_shards) = client.epoch_manager.num_shards(&head.epoch_id) {
             for shard_id in 0..num_shards {
-                let tracked = match me {
-                    None => false,
-                    Some(me) => client
-                        .epoch_manager
-                        .cares_about_shard_from_prev_block(&head.last_block_hash, me, shard_id)
-                        .unwrap_or(false),
-                };
+                let tracked = client.shard_tracker.care_about_shard(
```

Comment:
Just for my own education. 
`care_about_shard` is what we are using to actually track shards, right? This is THE place where we smoosh together tracked_config and EpochInfo, and decide what to do if node is a validator with some unusual configuration?

Reply:
Yes, right.
`EpochManager::cares_about_shard_from_prev_block()` doesn't take into account `tracked_shards` and `tracked_accounts`.